docs/miniCore.md

```diff
@@ -0,0 +1,194 @@
+I don't want to go into more detail to not annoy anybody, but I will say this much: among other benefits 
+it caters for the fundamental problem that neither RDF standard reification nor RDF-star are able to 
+solve. It bridges the gap between a statement and its annotation unambiguously, and effortlessly, e.g.:
+```turtle
+# serialized as NNG
+[] { :s :p :o } :u :v .
+:A { :s :p :o } :x :y .
+# serialized as TriG
+_:NG  :u :v .
+:X  :x :y .
+_:NG { :s :p :o }
+:A { :s :p :o }
+```
+There is not separation between an assertion and its annotation. There is at most, in the TriG mapping, 
+an indirection. For more about nesting, compatability to installed uses of named graphs, ease of querying, 
+etc see the ilink above.
+
+
+## Standardizing RDF 1.2
+
+### Minimal Core
+
+I propose to include into RDF 1.2 only the annotation syntax (for Turtle, TriG, maybe an equivalent 
+in JSON-LD), together with support in SPARQL. The mapping to RDF standard reification normatively 
+describes the semantics. Supporting that mapping provides complete RDF 1.2 support.
+
+The reification quad is well disliked, but it provides a common denominator since RDF 1999.
+For stores that don’t plan to support lots of annotations or heavy loads of LPG-style data this still 
+offers an economical path to full RDF 1.2 compatability (of course, some stores even support RDF 
+standard reification *very* efficiently, despite its syntactic verbosity, so there’s no judgement 
+involved at all). For stores that implement RDF-star as named graphs this arrangement provides 
+security that their implementation won’t be invalidated by some unforseen usage of a new term type
+in RDF like the proposed RDF-star triple term.
+
+Explicit support for un-asserted triples has to go somewhere. RDF standard reification already is
+misused to support it. We could strengthen and rectify that misuse by supporting it with a proper
+subtype of `rdf:Statement`. That would again require a (pretty trivial) mapping to triple terms in 
+RDF-star (and is provided in NNG already).
+
+According to this proposal it would be NORMATIVE that a system claiming to support RDF 1.2 has to 
+support the annotation shorthand syntax *at least* via RDF standard reification. However, any 
+implementation would be free to do whatever it likes under the hood to more efficiently store 
+the annotation syntax. Proper extensions to that effect may be defined.
+
+
+### Extensions
+
+#### RDF-star
+
+Of course getting rid of the RDF standard reification quad was *the* motivation for the RDF-star effort. 
+However, it requires a considerable extension to the core of RDF and is still struggling in a lot
+of crucial details. I propose therefore to make it a semantic extension to RDF - endorsed by this WG,
+but still separate and optional. A store deciding to support RDF-star would be welcome to map the 
+annotation syntax to triple terms, not to standard reification quads. 
```

Comment:
There are two things that do not quite make sense in this sentence, with respect to how semantic extensions are defined in RDF Semantics [1]:

* semantic extensions do not augment the abstract syntax, so if triple terms are not part of the abstract syntax, there is no way for a semantic extension to map anything to triple terms;

* the semantics of RDF is defined purely on the abstract syntax, it has no access to the concrete syntax. Since annotation are only syntactic sugar in the concrete syntax and have no counterpart in the abstract syntax, semantic extensions can not say anything about annotations.

What could be done, however, would be to define a semantic extension where ill-formed instances of rdf:Statements (e.g. statements with 0 or 2 subjects) are forbidden by the *syntactic restrictions* of this extension. Statements would still be regular IRIs or bnodes, not triple terms, but at least they would be guaranteed to always have a subject, a predictae and an object (and only one of each).

[1] https://www.w3.org/TR/rdf11-semantics/#dfn-semantic-extension

Reply:
I see, so introducing a new term into the model is not a semantic extension anymore. IIUC, a  syntax that supports syntactic sugar for reification could be accompanied by a semantic extension as you outline it. I don't follow why some insist that the model needs to be extended with a new term type. IMO such a syntax, and syntactic sugar like triple terms for N-Triples, and clearly defined restrictions on the mapping to RDF standard refication, would provide enough common ground and would leave implementations free to choose if they implement it via RDF standard reification or RDF-star or named graphs or whatever.